wsd/RequestDetails.hpp

```diff
@@ -133,6 +133,12 @@ class RequestDetails
     /// Decode and sanitize a URI.
     static Poco::URI sanitizeURI(const std::string& uri);
 
+    // if request uri is an alias, replace request uri host and port with
+    // original hostname and port defined by aliases tag from coolwsd.xml
+    // to avoid possibility of opening the same file as two if the WOPI host
+    // is accessed using different aliases
+    static void setUriHostAndPort(std::string& uriHost, std::string& uriPort);
```

Comment:
`getUriHostAndPort` is more reasonable, since this is _reading_ the host and port.